isis/src/base/apps/isisimport/isisimport.cpp

```diff
@@ -43,6 +46,10 @@ namespace Isis {
       try {
         // try to convert pvl to json
         jsonData = pvlToJSON(inputFileName.toString());
+        // QString labelOffset = jsonData["^QUBE"]["Value"];
+        if (jsonData.find("^QUBE") != jsonData.end()) {
+          jsonData["imageOffset"] = jsonData["^QUBE"]["Value"];
+        }
```

Comment:
This won't work. PvltoJSON now converts ^ to ptr. This can be moved to translation object using {{ ptrQUBE.Value }}